### DISCUSSION TITLE
Multiple weather stations

### DISCUSSION THREAD

**Discussion by j42z** [cytech/Home-Assistant-wundergroundpws#136] [Q&A]
Hi,

I have two weather stations in different locations. How can I add more than one pws_id in the new configuration format?

**Comment by BuurmanG** [comment]
Happy new year! 

I have the same topic: Three weather stations to monitor.... I need an example to adjust this in configuration.yaml

**Comment by cytech** [comment]
v1.x.x does not support this.
if you need this and had it working before, then revert back to version v0.8.3, without weather card support

**Comment by gregdcBe** [comment]
Hello @cytech ,
First, thanks for great job done for wunderground integration.

Is this multiple stations feature planned in coming releases ?
I also need to monitor multiple stations.

Thanks,

Greg

**Comment by cytech** [comment]
v2.x.x supports multiple
https://github.com/cytech/Home-Assistant-wundergroundpws/tree/v2.X.X

make sure you read the readme if you plan on changing to v2.x.x
https://github.com/cytech/Home-Assistant-wundergroundpws/tree/v2.X.X#upgrade

it is NOT an upgrade to v1

**Reply by BuurmanG** [reply]
Great Job

**Reply by gregdcBe** [reply]
Thanks for this, I'll proceed with installation of v2.

**Comment by cytech** [comment]
closed as outdated

**Comment by cytech** [comment]
closed as outdated
